metadata.json

```diff
@@ -42,6 +42,10 @@
       "name": "puppet/extlib",
       "version_requirement": ">= 3.0.0 < 7.0.0"
     },
+    {
+      "name": "puppet/mosquitto",
+      "version_requirement": ">= 1.0.1 < 2.0.0"
+    },
```

Comment:
I wonder if this should be a hard dependency or rather a soft one. It's really an edge case so most users won't need it.

Reply:
If it's a soft dependency, it would need to be included explicitly in the installer Puppetfile?

Reply:
With the soft dependency approach (what I have now implemented) what is the proper method to test?

Reply:
add it to [`.fixtures.yml`](https://github.com/theforeman/puppet-foreman_proxy/blob/master/.fixtures.yml)